oshi-core/src/test/java/oshi/hardware/DefaultAbstractGlobalMemory.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Oshi (https://github.com/dblock/oshi)
+ *
+ * Copyright (c) 2010 - 2017 The Oshi Project Team
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Maintainers:
+ * dblock[at]dblock[dot]org
+ * widdis[at]gmail[dot]com
+ * enrico.bianchi[at]gmail[dot]com
+ *
+ * Contributors:
+ * https://github.com/dblock/oshi/graphs/contributors
+ */
+
+package oshi.hardware;
+
+import oshi.hardware.common.AbstractGlobalMemory;
+
+public class DefaultAbstractGlobalMemory extends AbstractGlobalMemory {
```

Comment:
I wonder if setters have any sense in GlobalMemory.java because the values (swap, available, etcc) will always be overriden when calling getters. For example, setAvailable will be ignored since getAvailable call the updateMeminfo() which override the memAvailable property